webcompat/static/css/development/components/search-bar.css

```diff
@@ -39,7 +39,12 @@
       font-size: var(--SearchBar-fontSize);
       flex: 1;
       margin: 0 1em;
-      background-color: transparent;
+      background-color: white;
```

Comment:
@deepthivenkat you can add a new variable in this section: https://github.com/webcompat/webcompat.com/blob/master/webcompat/static/css/development/components/search-bar.css#L4